src/list.js

```diff
@@ -0,0 +1,26 @@
+const bestScores = [
+  {
+    name: 'Alejandro',
+    score: 100,
+  },
+  {
+    name: 'Matias',
+    score: 200,
+  },
+  {
+    name: 'Katia',
+    score: 150,
+  },
+  {
+    name: 'Janeth',
+    score: 500,
+  },
+];
+
+export default function fillList() {
```

Comment:
- [OPTIONAL] For proper ES6 syntax, let use of arrow functions instead of the function keyword.

Reply:
Hi @Mihndim2020 thank you very much for your suggestion and approval of my project I already committed the change. 🤓  👨‍💻